app/src/app/views/stix/stix-dialog/stix-dialog.component.html

```diff
@@ -4,6 +4,7 @@
             <div class="stix-content" *ngIf="!validating">
                 <mat-toolbar>
                     <button mat-button mat-dialog-close class="control" matTooltip="close"><mat-icon aria-label="close">close</mat-icon></button>
+                    <button *ngIf="prevObject" mat-button class="control" matTooltip="back" [disabled]="editing" (click)="goBack()"><mat-icon aria-label="back">arrow_back</mat-icon></button>
```

Comment:
Navigation icon should be before the close icon, and/or maybe the close icon shouldn't be available when there is a page to go back to?

https://material.io/components/app-bars-top#anatomy